css/index.css

```
@@ -1,3 +1,15 @@
+.navbar {
+    display: grid;
+    background-color: #C7522A;
+    justify-content: space-between;
+    padding-left: 41%;
+    padding: 3%;
+    position: sticky;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto;
+    list-style-type: none;
+    text-decoration: none;
+}
```

```suggestion
}
.navbar a {
    text-decoration: none;
}
```

> I am trying to remove the underlines in my hyperlinks in the navbar. I believe I'm using text.decoration = 'none' but I don't see a change.

The problem is that it's not the `navbar` that is being underlined... it's the default style for `a` elements inside, so to override that, you need a more precise pattern.